### DISCUSSION TITLE
The sudden drop of temperature probe and thickness probe in the spread of solid fire

### DISCUSSION THREAD

**Discussion by uihiuy** [firemodels/fds#13824] [General]
I'm simulating a solid fire spread but the output data shows that the temperature drops, the thickness drops drops suddenly, etc. What's the reason for this? Is the grid burning out or what?
[新建文本文档.txt](https://github.com/user-attachments/files/17990485/default.txt)
![image](https://github.com/user-attachments/assets/c24daf6c-57a2-43e4-8336-cbcd80e13855)

**Comment by drjfloyd** [comment]
You have specified inputs that will let the PMMA pyrolyze. The thickness is decreasing because the solid is pyrolyzing to vapor. WALL TEMPERATURE has nothing to output when the solid disappears and at that point just shows TMPA.

**Reply by uihiuy** [reply]
How to make the temperature rise curve more continuous rather than suddenly broken, how to set it up, thank you

**Reply by drjfloyd** [reply]
Predicting fire growth and spread puts you in the realm of a research application of FDS. We cannot do your research for you. You need to look at the outputs of your current simulation, understand what is happening (add more outputs and rerun if needed), decide if the behavior is reasonable or not, and then based on that make any adjustments to inputs that you feel are justified for your specific application. 

For example have you:

Looked at the heat flux your ignition source creates? Is it reasonable? 
Run a virtual TGA to understand the behavior of your reaction kinetics? 
Done a hand calculation to see how quicky you expect your ignition source to heat the material to the point where it pyrolyzes?
Done a hand calculation to estimate how quickly your material will burnaway once it pyrolyzes given the incident flux?

**Reply by drjfloyd** [reply]
You should also keep in mind that this input means a grid cell with more than one exposed face can burn on more than one side. Make sure you understand how BURN_AWAY works. Study at the examples in the User's Guide and Verification Guide.

&OBST XB = 0.005 0.055 0.005 0.085 0 0.005
      SURF_ID = 'PMMA CUBE'
      COLOR='PURPLE' /
